user/pages/03.commerce2/02.developer-guide/12.recurring/02.getting-started/docs.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Getting started
+taxonomy:
+    category: docs
+---
+
+## Setting up subscriptions: First steps
+### Step 1 - Create/enable a payment method
+Each subscription has a Payment method, so you'll need to set up at least one enabled payment gateway that can be used by a customer to create a payment method.
```

Comment:
Our docs have a list of gateways with an onsite/offsite column. Since offsite gateways can't store payment methods right now, that means that people can refer to the list of onsite gateways.